src/main/java/step3/domain/LottoTicket.java

```diff
@@ -1,15 +1,23 @@
 package step3.domain;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class LottoTicket {
-    private List<List<Integer>> bundle;
-
-    public LottoTicket(List<List<Integer>> bundle) {
-        this.bundle = bundle;
-    }
+    private List<List<Integer>> bundle = new ArrayList<>();
 
     public List<List<Integer>> getBundle() {
         return bundle;
     }
+
+    public void addLotto(List<Integer> lottoNumbers) {
+        System.out.println(lottoNumbers);
+        bundle.add(lottoNumbers);
+    }
+
+    public void addBundle(List<List<Integer>> lottoBundle) {
+        for (List<Integer> lotto: lottoBundle) {
+            bundle.add(lotto);
+        }
```

Comment:
```suggestion
        bundle.addAll(lottoBundle);
```

addAll을 활용해볼 수 있겠네요!

Reply:
@_@ 좋은 정보 감사합니다!!